### PR TITLE
Sort service names in the UI

### PR DESCRIPTION
**Pull request by zeagord**
Implementation for #2204

### CODE REVIEW

File: zipkin-ui/js/component_ui/serviceName.js [hunk 1]
```diff
@@ -18,7 +18,12 @@ export default component(function serviceName() {
   this.updateServiceNameDropdown = function(ev, data) {
     $('#serviceName').empty();
     this.$node.append($($.parseHTML('<option value="all">all</option>')));
-
+    // Sorting based on the localCompare so that sorting can be accomplished for non-ascii (non english) service names.
+    if (data.names) {
```
Comment (codefromthecrypt) [inline zipkin-ui/js/component_ui/serviceName.js]: now... is there a place for a unit test? :D

Comment (zeagord) [reply zipkin-ui/js/component_ui/serviceName.js]: Added them :)

**Comment by codefromthecrypt** [comment]
npm lint hates you but I ❤️ you

**Comment by zeagord** [comment]
I am so sorry. Let me correct it. Wondering how my editor missed it.

**Pull request merged**